frontend/src/store/reducers/app.ts

```diff
@@ -7,8 +7,18 @@ import {
 } from 'types/actions/app';
 import InitialValueTypes from 'types/reducer/app';
 
+const isDarkTheme = (): boolean => {
+	const initialTheme = localStorage.getItem('theme');
```

Comment:
also can we use local storage browser api here under `api/browser/localstorage/get`